src/Graph.h

```diff
@@ -116,26 +116,62 @@ class Graph
     unsigned int getSize();
 
     /**
+     * Dijkstra method
      * findPathLengths() will get all the path lengths with a certain node from the center.
      * @param start the start node
      * @param metric an integer specifying the data value that we want to create the minimum
-     * paths with. Metric is 0 for fare (cost), 1 for miles (length), and 2 for time.
+     * paths with. Metric is 0 for fare (cost), 1 for time, and 2 for miles (length)
      * @return a vector of pairs of nodes and the length from the start node.
      * The length from the start node to itself is 0.
      */
     std::vector<std::pair<Node*, double>> findPathLengths(Node* start, int metric);
```

Comment:
this is just curiousity, but why an int instead of unsigned?

Reply:
I just made it an int lol